### PR TITLE
Add JSX Newline eslint rule and format code

### PR DESCRIPTION
**Pull request by michalszorad**
Added new rule `'react/jsx-newline': ['warn', {prevent: false}],` to add extra empty line between elements
Screenshot 2023-01-30 at 14 29 44

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ module.exports = {
     'react-native/no-unused-styles': 2,
     'react-native/split-platform-components': 0,
     'react/display-name': 0,
+    'react/jsx-newline': ['warn', {prevent: false}],
```
Comment (michalszorad) [inline .eslintrc.js:59]: new eslint rule

File: src/Send/ConfirmScreen/ConfirmScreen.tsx [hunk 2]
```diff
@@ -107,10 +107,10 @@ const Fees = ({yoroiUnsignedTx}: {yoroiUnsignedTx: YoroiUnsignedTx}) => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
   const feeAmount = Amounts.getAmount(yoroiUnsignedTx.fee, wallet.primaryToken.identifier)
-
+  const text = `${strings.fees}: ${formatTokenWithSymbol(new BigNumber(feeAmount.quantity), wallet.primaryToken)}`
```
Comment (michalszorad) [inline src/Send/ConfirmScreen/ConfirmScreen.tsx:110]: eslint was complaining, had to put the text into a variable

File: src/Send/ConfirmScreen/ConfirmScreen.tsx [hunk 3]
```diff
@@ -129,10 +129,13 @@ const BalanceAfter = ({yoroiUnsignedTx}: {yoroiUnsignedTx: YoroiUnsignedTx}) =>
     ]),
   )
   const primaryAmountAfter = Amounts.getAmount(balancesAfter, wallet.primaryToken.identifier)
-
+  const text = `${strings.balanceAfterTx}: ${formatTokenWithSymbol(
```
Comment (michalszorad) [inline src/Send/ConfirmScreen/ConfirmScreen.tsx:132]: eslint was complaining, had to put the text into a variable

File: src/Staking/PoolWarningModal/PoolWarningModal.tsx [hunk 4]
```diff
@@ -70,11 +75,8 @@ type BulletPointProps = {
 }
 
 const BulletPointItem = ({textRow, style}: BulletPointProps) => {
-  return (
-    <Text style={style}>
-      {'\u2022'} {textRow}
-    </Text>
-  )
+  const text = `${'\u2022'} ${textRow}`
```
Comment (michalszorad) [inline src/Staking/PoolWarningModal/PoolWarningModal.tsx:78]: eslint was complaining, had to put the text into a variable

File: src/TxHistory/BalanceBanner.tsx [hunk 5]
```diff
@@ -105,10 +105,10 @@ const PairedBalance = ({privacyMode}: {privacyMode: boolean}) => {
     config.decimals,
   )
   const pairedTotal = privacyMode ? hiddenPairedTotal : secondaryExchangeQuantity
-
+  const text = `${pairedTotal} ${currency}`
```
Comment (michalszorad) [inline src/TxHistory/BalanceBanner.tsx:108]: eslint was complaining, had to put the text into a variable

File: src/TxHistory/TxDetails/TxDetails.tsx [hunk 6]
```diff
@@ -171,11 +183,8 @@ const Fee = ({amount}: {amount: BigNumber}) => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
 
-  return (
-    <Text small>
-      {strings.fee} {formatTokenWithSymbol(amount, wallet.primaryToken)}
-    </Text>
-  )
+  const text = `${strings.fee} ${formatTokenWithSymbol(amount, wallet.primaryToken)}`
```
Comment (michalszorad) [inline src/TxHistory/TxDetails/TxDetails.tsx:186]: eslint was complaining, had to put the text into a variable

File: src/TxHistory/TxDetails/TxDetails.tsx [hunk 7]
```diff
@@ -199,10 +210,11 @@ type AddressEntryProps = {
   showModalForAddress: (string) => void
 }
 const AddressEntry = ({address, path, isHighlighted, showModalForAddress}: AddressEntryProps) => {
+  const text = `(${path}) ${address}`
```
Comment (michalszorad) [inline src/TxHistory/TxDetails/TxDetails.tsx:213]: eslint was complaining, had to put the text into a variable

**Review by wolverineks — APPROVED** [review]

**Review by stackchain — APPROVED** [review]

**Pull request merged**